### PR TITLE
Update opentok.gemspec

### PR DESCRIPTION
**Pull request by Lucashuang0802**
Attempt to change version semantics to see if it passes. In response to the issue #148.

### CODE REVIEW

File: opentok.gemspec [hunk 1]
```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   # spec.add_development_dependency "debugger", "~> 1.6.6"
 
   spec.add_dependency "addressable", "~> 2.3" #  2.3.0 <= version < 3.0.0
-  spec.add_dependency "httparty", "~> 0.14.0"
+  spec.add_dependency "httparty", ">= 0.14.0"
```
Comment (aiham) [inline opentok.gemspec]: @Lucashuang0802 Why not set it to `~> 0.15.5`?

Comment (Tumas) [reply opentok.gemspec]: Hey, can we get any activity on this?

Comment (thepatrick) [reply opentok.gemspec]: @aiham 0.15.5 requires ruby v2 or higher

**Comment by thepatrick** [comment]
Will address this with #151 instead

**Pull request closed without merging**